client/src/components/AlumniWorkspace.js

```diff
@@ -11,13 +16,68 @@ export default class AlumniWorkspace extends Component {
         super(props)
         this.state = {
             activeItem: 'collegesAccepted',
-            collegesAccepted: []
+            collegesAccepted: [],
+            
         }
     }
 
     handleMenuClick = (e, { id }) => this.setState({ activeItem: id })
 
-    render() {
+    componentWillMount(){
+        this.getCollegesAcceptedInto();
+    }
+
+    getCollegesAcceptedInto = () => {
+        const theEndPoint = `/alumni/collegesAcceptedInto/all/${this.props.userDetails._id}`;
+        makeCall({}, theEndPoint, `GET`)
+        .then(res => {
+            this.setState({
+                collegesAccepted: res
+            })
+        })
+        .catch(e => {
+            console.log('Error #getCollegeAcceptedInto', e)
+        })
+    }
+
+    renderCollegesAcceptedInto = () => {
+        let colleges = this.state.collegesAccepted;
+        if(this.state.collegeAcceptedInto == [] || this.state.collegesAccepted == null){
+            return(
+                <Message info>
+                    <Message.Header>No colleges in accepted college list.</Message.Header>
+                    {
+                        <Message.Content>Please add any colleges you have been offered an admission into </Message.Content>
```

Comment:
nit: this feel like an unfinished sentence. maybe because of ending a sentence with a preposition with a full stop? maybe "colleges that have offered you admission" or "colleges you acceptance offers from". Idk. Don't have especially string thoughts on the language here outside of the first sentence.